src/Commands/Command.php

```diff
@@ -61,7 +61,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // Configure extra output formats
         $this->output->getFormatter()->setStyle('bold', new OutputFormatterStyle('blue'));
 
-        return $this->fire();
+        $ret = $this->fire();
+        // returning a default value of 1 here to satisfy
+        // symfony/console/Command/Command.php::run()
+        return is_int($ret) ? $ret : 1;
```

Comment:
We should return one of the constants defined on `Command` - and for that matter we should probably just get `fire()` to return the appropriate const based on whether it was successful or not.

Reply:
Have changed the default `1` to Console\Command\Command::SUCCESS

I don't think updating the 17 fire() methods is worth it, if we did this we'd also want to return `Console\Comman\Command::FAILURE` everytime there's some sort of error, so we'd need to update lots of code.

I really don't think slightly nicer output colours (I'm presuming that's what the outcome) is worth the effort.

Reply:
Yeah, fair enough. I think cow is a mangled enough mess that the benefits aren't really beneficial without a considerable refactor.